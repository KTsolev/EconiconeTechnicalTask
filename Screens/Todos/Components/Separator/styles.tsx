import {StyleSheet} from 'react-native';

export const styles = StyleSheet.create({
  separator: {
    width: '100%',
    marginVertical: 8,
    height: 2,
    backgroundColor: '#c6c6c6c6',
  },
});
