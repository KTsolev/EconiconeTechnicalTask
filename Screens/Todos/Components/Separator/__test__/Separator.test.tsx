import React from 'react';
import {Separator} from '../Separator';
import renderer from 'react-test-renderer';

describe('AddNewItem tests', () => {
  it('should render component', async () => {
    const tree = renderer.create(<Separator />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
