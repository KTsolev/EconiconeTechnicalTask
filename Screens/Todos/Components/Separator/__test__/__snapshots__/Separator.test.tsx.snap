// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddNewItem tests should render component 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#c6c6c6c6",
      "height": 2,
      "marginVertical": 8,
      "width": "100%",
    }
  }
/>
`;
